dataprofiler/profilers/json_decoder.py

```diff
@@ -3,12 +3,14 @@
 
 if TYPE_CHECKING:
     from .base_column_profilers import BaseColumnProfiler
-    from .profilers import column_profile_compilers as col_pro_compilers
+    from .column_profile_compilers import BaseCompiler
```

Comment:
not sure if how `.profilers` works or why we can't have `.column_profile_compilers`

Reply:
can you explain more? why was this changed?

Reply:
this seems more straight forward. I also don't know where `.profilers` references to

Reply:
ahh, yes that's weird

Reply:
`from . import column_profile_compilers as col_pro_compilers` would keep consistency